src/test/resources/db/booking/clear-tables.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM bookings;
+DELETE FROM accommodations;
+DELETE FROM addresses;
```

Comment:
add emty line in all of your .sql scripts

Reply:
approve please, empty lines added